processes/ingest/hathi_trust.py

```diff
@@ -99,23 +107,14 @@ def importFromHathiFile(self, hathiURL):
             self.readHathiFile(hathiTSV)
 
     def readHathiFile(self, hathiTSV):
-        processed_record_count = 0
-
-        while True:
-            if self.ingest_limit and processed_record_count >= self.ingest_limit:
+        for number_of_books_ingested, book in enumerate(hathiTSV):
+            if self.ingest_limit and number_of_books_ingested > self.ingest_limit:
                 break
 
-            try:
-                row = next(hathiTSV)
-                logger.info('Reading row')
-            except csv.Error as e:
-                logger.warning('Unable to read TSV row')
-                logger.debug(e)
-                continue
-            except StopIteration:
-                logger.info('Reached end of TSV file')
-                break
+            book_right = book[2]
+            book_date_updated = book[14]
+            start_date_time = datetime.now(timezone.utc).replace(tzinfo=None) - timedelta(days=1) 
+            hathi_date_modified = datetime.strptime(book[14],'%Y-%m-%d %H:%M:%S')
```

Comment:
Should we use our function here? 

`returnHathiDateFormat`

Reply:
The 'updated dates' field for the hathi data rows usually have timestamps in this format 2017-02-16 05:58:14 which is a timestamp format of '%Y-%m-%d %H:%M:%S' that doesn't match any format in the current `returnHathiDateFormatMethod`. This is why I manually placed the timestamp format in the datetime object.

Reply:
Copy!